### ISSUE TITLE
Call of Duty 3 crashing RPCS3

### ISSUE THREAD

**Issue by tarakbumba** [RPCS3/rpcs3#16380]
### Quick summary

I used to play Call of Duty 3 with minor artifacts but now it completely crashes RPCS3.

### Details

I used to play Call of Duty 3 with minor artifacts but now it completely crashes RPCS3. I use nightly builds. My machine is a fully updated Windows 11 24H2 with Rtx4060 supplied HP Notebook.

### Attach a log file

[RPCS3.log](https://github.com/user-attachments/files/18050104/RPCS3.log)

**Comment by crackersden** [comment]
i'm having exactly the same issue, everything goes fine until you reach the part where you have to go through a tunnel (early on)
tried it with fifo accuracy on atomic and drive wake up delay 200 and 400 and it still crashes.
it would be nice to have more feedback from other users

**Comment by OneShot95** [comment]
Also happening to me, sometime the game makes it a few minutes. Flickering AI the all textures disappear and then crashes.

**Comment by digant73** [comment]
is the issue present on 0.0.32-16658?

**Comment by tarakbumba** [comment]
> is the issue present on 0.0.32-16658?

It seems to me that issue is gone. I can successfully played about 30 min without crash. But keep in mind that there are often flickering graphical glitches present.

**Comment by digant73** [comment]
Recent rpcs3 build probably fixed the crash while old 0.0.32-16658 should avoid the graphical issues. Can you confirm please?

**Comment by tarakbumba** [comment]
> Recent rpcs3 build probably fixed the crash while old 0.0.32-16658 should avoid the graphical issues. Can you confirm please?

I downloaded 0.0.32-16658 version and it locks up game and then crash. Sorry.

**Comment by digant73** [comment]
yes the instability is due to another bug. That old build is not affected by the graphical issue

**Comment by digant73** [comment]
@tarakbumba is the issue fixed on recent build?

**Comment by tarakbumba** [comment]
I'm really sorry that I totally forgot this bug report. It works on recent builds with glitches here and there. But I haven't encounter any crash yet.

**Comment by digant73** [comment]
> I'm really sorry that I totally forgot this bug report. It works on recent builds with glitches here and there. But I haven't encounter any crash yet.

good. Eventually open a new ticket with the details for the glitches you see. Which glitches do you see? are new glitches or they were present also in the past?
